### DISCUSSION TITLE
Github support

### DISCUSSION THREAD

**Discussion by alxndr-stlz** [community/community#63601] [GitHub Education]
Hello friends,

Im currently waiting since 65 days for a reply to my problem with GitHub education.

Anyone else experiencing similar issues?

I've tried reaching out several times, but didn't get an answer.

Once I've received an email, which stated, that the GitHub support is experiencing high volume of requests and that the response times can be a little longer. But I don't think, that they've meant 65 days.

So yeah, i've wanted to reach out another way - so here I am.

Kindly,
Alex

**Comment by samus-aran** [comment]
I'll be closing this discussion due to lack of recent activity. If you still haven't received your Global Campus benefits please feel free to re-open.
